packages/Manager/build.gradle.kts

```diff
@@ -166,3 +183,39 @@ testlogger {
   showSkippedStandardStreams = false
   showFailedStandardStreams = true
 }
+
+tasks.register("loadSampleData") {
+  dependsOn("flywayMigrate")
+  doLast {
+    println("Loading Sample Data")
+    SingleConnectionDataSource(
+            dbConfig["devUrl"]!!, dbConfig["user"]!!, dbConfig["password"]!!, true)
+        .let {
+          it.connection.use { connection ->
+            executeSqlScript(
+                connection, FileSystemResource("./sample-data/allocation_data.sql"))
+
+            executeSqlScript(
+                connection, EncodedResource(FileSystemResource("./sample-data/observation_data_function.sql")), false, false, DEFAULT_COMMENT_PREFIX, EOF_STATEMENT_SEPARATOR1,
+              DEFAULT_BLOCK_COMMENT_START_DELIMITER, DEFAULT_BLOCK_COMMENT_END_DELIMITER
+            )
+            executeSqlScript(
+                connection, FileSystemResource("./sample-data/observation_data.sql"))
+          }
+        }
+  }
+}
+
+tasks.register("refreshSampleData") {
+  dependsOn("flywayMigrate")
+  doLast { println("refreshSampleData.")
+
+    SingleConnectionDataSource(
+      dbConfig["devUrl"]!!, dbConfig["user"]!!, dbConfig["password"]!!, true)
+      .let {
+        JdbcTemplate(it).let {
+          it.execute("UPDATE observations SET observed_at = observed_at + INTERVAL '1 YEAR';")
```

Comment:
If I understand correctly, this shifts all the data one year forward right? Should we try and make this relative to now? E.g. get the difference between now and max(observed_at), and add that (plus _n_ if we want to extend into the future) to all all observed_at?

Reply:
Done